users/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+
+from .models import Followers, User
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = [
+            "username",
+            "first_name",
+            "last_name",
+            "description",
+            "date_of_birth",
```

Comment:
Probably users don't want `date_of_birth` to be publicly accessable. So let's have at least two serializers for the User model: 
1. Serializer containing only public user data
2. Serializer containing full user data

They can subclass one another

Reply:
Done